README.md

```diff
@@ -49,6 +49,67 @@ fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
 
 See the [examples](./examples) directory for different integration patterns.
 
+## Ecosystem
+
+### Related Crates
+
+In addition to `opentelemetry`, the [`open-telemetry/opentelemetry-rust`]
+repository contains several additional crates designed to be used with the
+`opentelemetry` ecosystem. This includes a collection of trace `SpanExporter`
+and metrics pull and push controller implementations, as well as utility and
+adapter crates to assist in propagating state and instrumenting applications.
+
+In particular, the following crates are likely to be of interest:
+
+- [`opentelemetry-jaeger`] provides a pipeline and exporter for sending trace
+  information to [`Jaeger`].
+- [`opentelemetry-otlp`] exporter for sending trace and metric data in the OTLP
+  format to the OpenTelemetry collector.
+- [`opentelemetry-prometheus`] provides a pipeline and exporter for sending
+  metrics information to [`Prometheus`].
+- [`opentelemetry-zipkin`] provides a pipeline and exporter for sending trace
+  information to [`Zipkin`].
+- [`opentelemetry-contrib`] provides additional exporters to vendors like
+  [`Datadog`].
+- [`opentelemetry-semantic-conventions`] provides standard names and semantic
+  otel conventions.
+
+Additionally, there are also several third-party crates which are not
+maintained by the `opentelemetry` project. These include:
+
+- [`tracing-opentelemetry`] provides integration for applications instrumented
+  using the [`tracing`] API and ecosystem.
+- [`actix-web-opentelemetry`] provides integration for the [`actix-web`] web
+  server and ecosystem.
+- [`opentelemetry-application-insights`] provides an unofficial [Azure
+  Application Insights] exporter.
+- [`opentelemetry-tide`] provides integration for the [`Tide`] web server and
+  ecosystem.
+
```

Comment:
Looks nice! I think there is another one https://github.com/vivint-smarthome/opentelemetry-stackdriver. Although it doesn't seem to be in active development.

Reply:
Yeah I didn't want to link to projects that seemed unmaintained or really out of date as that is probably a bad experience. Could do a PR to get those up to date possibly and then link them here